src/components/RSVPform.js

```diff
@@ -0,0 +1,175 @@
+import React, {useState} from "react";
+import {Form, Button, Col, FormControl, InputGroup} from "react-bootstrap";
+import {db} from "../firebase";
+
+const RSVPform = () => {
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [rsvp, setRsvp] = useState("");
+  const [plusOne, setPlusOne] = useState(false);
+  const [plusFName, setPlusFName] = useState("");
+  const [plusLName, setPlusLName] = useState("");
+  const [allergies, setAllergies] = useState("");
+  const [guestEmail, setEmail] = useState("");
+
+  const addUser = () => {
+    const data = {
+      firstName: firstName,
+      lastName: lastName,
+      uid: new Date().getTime(),
+      rsvp: rsvp,
+      plusOne: plusOne,
+      plusFName: plusFName,
+      plusLName: plusLName,
+      allergies: allergies,
+      email: guestEmail,
+    };
+    db.collection("guests")
```

Comment:
I'd prefer to see this as part of the db API. So your components don't need to know anything about how firebase/the db works. It would be nice to see `db` export something like

```
export const addUser = (user) => { ... }
```

which you just call directly in your component

Reply:
New file hierarchy: src > firebase > db.js.

Right now it only has the addUser function, I think another one I may try to build is update info -- but that may be more down the line...